src/game.cc

```diff
@@ -1348,8 +1348,16 @@ static int gameDbInit()
         return -1;
     }
 
+    // SFALL: custom patch file name.
+    char* patch_filename = nullptr;
+    if (configGetString(&gSfallConfig, SFALL_CONFIG_MISC_KEY, SFALL_CONFIG_PATCH_FILE, &patch_filename)) {
+        if (patch_filename == nullptr || *patch_file_name == '\0') {
+            patch_filename = "patch%03d.dat";
+        }
+    }
+
     for (patch_index = 0; patch_index < 1000; patch_index++) {
-        snprintf(filename, sizeof(filename), "patch%03d.dat", patch_index);
+        snprintf(filename, sizeof(filename), patch_filename, patch_index);
```

Comment:
Do we really need this option? Any mod relies on it? We have a much better mods system in sfall 4.x. All this patch stuff is relics from early modding years.

Reply:
Patches is still vanilla engine feature, so i guess 'll be better have to stay that thing.

Reply:
I'm talking about patch stuff added by sfall, that nobody uses now.